portfolio/src/main/webapp/script.js

```diff
@@ -26,4 +26,20 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
-//console.log('hiii');
\ No newline at end of file
+//console.log('hiii');
```

Comment:
Though it's commented out, we don't want to pollute our code with debugging logging statements when we commit to master.